tools/task-runner/runner

```diff
@@ -169,7 +169,7 @@ listrify(taskModules).run({
     stdouts: [],
 }).catch((e) => {
     // something went wrong, so log whatever we have
-    if (!e.stderr && !e.stdout) console.log(e);
+    if (!e.stderr && !e.stdout) console.log(e.message);
```

Comment:
careful with this, i found that tools are not consistent with how they return errors. maybe `e.message || e`?